### ISSUE TITLE
Can't get addon to work

### ISSUE THREAD

**Issue by robmonte** [tmarenko/twitch_chat_antiban#3]
When I install the addon, the stream page ends up loading a blank chat window. I tried completely turning off uBlock Origin in case that was blocking something and it still was not working.

I saw in the code it is using the nightdev chat URL and when I open that up for a specific channel directly, it does work and show chat.

&nbsp;

Edit: The popout chat DOES work for me, so it's something about replacing the banned chat page on my end.

If I inspect the chat element in the twitch window vs the popout chat, I can see that the div "chat_box" is empty on the webpage while it has content in the popout chat.

Twitch.tv page on the left, popout chat on the right:
![image](https://user-images.githubusercontent.com/17119716/123859582-7150ce00-d8ea-11eb-8176-dc979c0da8e7.png)

**Comment by robmonte** [comment]
No idea why but it's suddenly working for me today now. I didn't change anything but I had tried many times yesterday without success. Maybe some kind of loading race condition?  Feel free to close the issue if you think it isn't worth investigating.

**Comment by tmarenko** [comment]
> it's suddenly working for me today now

Is this related to issue https://github.com/tmarenko/twitch_chat_antiban/issues/4 ?

**Comment by robmonte** [comment]
> 
> 
> > it's suddenly working for me today now
> 
> Is this related to issue #4 ?

No, I had made two separate issues for the Alternate player request and then this issue.

This empty div issue was happening in the normal twitch page for me. It was happening for a couple hours when I first installed the addon even after many refreshes. Then I checked the page source and saw that empty div. After that, I gave up for a while and when I came back I opened a twitch channel I was banned on again and then suddenly the chat successfully loaded. Ever since then it has always worked for me now.
